docs/interfaces/usb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: USB
+tags:
+  - USB
+  - HMX
+  - C61
+  - MX-4
+  - MX-V
+---
+## Overview
+Our products has at least one usb port and many of them has a USB otg port where it is possible to connect keywords, usb memory sticks and connect it to a pc 'Ethernet over USB'.
+
+## Platform specific
+- [HMX](hmx/usb.md)
+- [MX-V](mxv/usb.md)
+- [C61](c61/usb.md)
+- [MX-4](mx4/usb.md)
+
+## Ethernet over USB
+Ethernet over USB is supported via RNDIS. Connect a USB-A cable to the
+USB0 port on the supported platforms and a free USB port on e.g. a PC, then power on
+the the unit (mx4 has possibility to change from host state to peripheral when powered) or reboot. This will give the PC the IP address 192.168.250.2
+and the MX-V 192.168.250.1.
```

Comment:
```suggestion
the unit or reboot. (The MX-4 has the capacity to switch from host to peripheral state when powered.) This will give the PC the IP address 192.168.250.2 and the MX-V 192.168.250.1.
```